ethers-etherscan/Cargo.toml

```diff
@@ -43,6 +43,7 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["rustls"]
+default = ["rustls", "solc"]
 openssl = ["reqwest/native-tls"]
 rustls = ["reqwest/rustls-tls"]
+solc = ["ethers-solc"]
```

Comment:
~~I don't think we need to add a name feature here, can use `ethers-solc` directly, I think.~~

hmm, I just realized `ethers` has `ethers-solc` as non-default, but if we enable it here by default then this won't have any effect.

so I think we should just keep it optional, disabled by default, and instead enable it in `ethers` main cargo toml if solc feature is enabled there

Reply:
I think `cfg(feature = "ethers-solc")` is enabled whenever the crate is built with the `ethers-solc` dependency so this should work